Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Ironhusk_Beetle.xml

```diff
@@ -20,13 +20,13 @@
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_IronhuskBeetle"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>20</ArmorRating_Blunt>
+					<ArmorRating_Blunt>42</ArmorRating_Blunt>

```

Comment:
Leave it alone. They've already got pretty good armor for something that's not a combat type in the first place.